core/server/src/main/java/alluxio/master/file/FileSystemMaster.java

```diff
@@ -313,6 +314,12 @@ public void processJournalEntry(JournalEntry entry) throws IOException {
       }
     } else if (entry.hasInodeDirectory()) {
       try {
+        // Add the directory to TTL buckets, the insert automatically rejects directory
+        // w/ Constants.NO_TTL
+        File.InodeDirectoryEntry inodeDirectoryEntry = entry.getInodeDirectory();
```

Comment:
replace `File.InodeDirectoryEntry` with just `InodeDirectory` ... this is for consistency with how other journal entry imports